crates/outbound-pg/src/lib.rs

```diff
@@ -285,3 +283,40 @@ where
         }
     });
 }
+
+struct PgNull;
```

Comment:
Can you add doc comments to this type explaining exactly what you explain in the PR description?